libs/perun/pipes/src/lib/member-status-tooltip.pipe.ts

```diff
@@ -8,8 +8,13 @@ import { parseMemberStatus } from '@perun-web-apps/perun/utils';
 export class MemberStatusTooltipPipe implements PipeTransform {
 
   transform(member: RichMember, showGroupStatuses: boolean): string {
-    const memberExpiration = member.memberAttributes.find(att => att.friendlyName === 'membershipExpiration');
-    const groupExpiration = member.memberAttributes.find(att => att.friendlyName === 'groupMembershipExpiration');
+    let memberExpiration = null;
+    let groupExpiration = null;
+
+    if(member.memberAttributes !== null){
+      memberExpiration = member.memberAttributes.find(att => att.friendlyName === 'membershipExpiration');
+      groupExpiration = member.memberAttributes.find(att => att.friendlyName === 'groupMembershipExpiration');
```

Comment:
Actually in this pipe and in the list it is showed the status on the group, but we dont care about group expiration there, we care about resource status, if the user is not expired on the resource
if this can be implemented you should make some research